spk/sickbeard-custom/src/conf/resource

```diff
@@ -0,0 +1,5 @@
+{
+	"port-config":{
+    	"protocol-file": "etc/sickbeard-custom.sc"
```

Comment:
I get install fail errors if I try to set this file to anything but within the `/var/package/{$package}/target/` realm.  I've tried `/var/packages/sickbeard-custom/scripts/sickbeard-custom.sc` and `../scripts/sickbeard-custom.sc`